tests/secondary_feature_tests.py

```diff
@@ -31,88 +35,98 @@ def setUp(self):
-        # test that acc works
-        ret0 = secondary.data_quality.data_quality(id=self.TEST_PARTICIPANT,
-                                           start=self.TEST_START_TIME_JERK,
-                                           end=self.TEST_START_TIME_JERK + self.MS_IN_DAY + 1,
-                                           resolution=self.MS_IN_DAY,
-                                           feature="accelerometer")
-        self.assertEqual(ret0['data'][0]['value'], 0.001412037037037037)
+#     def test_data_quality_acc(self):
+#         # test that acc works
+#         ret0 = secondary.data_quality.data_quality(id=self.TEST_PARTICIPANT,
+#                                            start=self.TEST_START_TIME_JERK,
+#                                            end=self.TEST_START_TIME_JERK + self.MS_IN_DAY + 1,
+#                                            resolution=self.MS_IN_DAY,
+#                                            feature="accelerometer")
+#         self.assertEqual(ret0['data'][0]['value'], 0.001412037037037037)
 
-    # 2. step count
-    def test_step_count_no_data(self):
-        # Test if the participant has no data
-        ret0 = secondary.step_count.step_count(id=self.TEST_PARTICIPANT,
-                                           start=self.TEST_END_TIME - 3 * self.MS_IN_DAY,
-                                           end=self.TEST_END_TIME,
-                                           resolution=self.MS_IN_DAY)
-        for x in ret0['data']:
-            self.assertEqual(x['value'], None)
+#     # 2. step count
+#     def test_step_count_no_data(self):
+#         # Test if the participant has no data
+#         ret0 = secondary.step_count.step_count(id=self.TEST_PARTICIPANT,
+#                                            start=self.TEST_END_TIME - 3 * self.MS_IN_DAY,
+#                                            end=self.TEST_END_TIME,
+#                                            resolution=self.MS_IN_DAY)
+#         for x in ret0['data']:
+#             self.assertEqual(x['value'], None)
 
-    def test_step_count(self):
-        # Test that step count works
-        ret0 = secondary.step_count.step_count(id=self.TEST_PARTICIPANT_STEPS,
+#     def test_step_count(self):
+#         # Test that step count works
+#         ret0 = secondary.step_count.step_count(id=self.TEST_PARTICIPANT_STEPS,
+#                                            start=self.TEST_START_TIME_STEPS,
+#                                            end=self.TEST_START_TIME_STEPS + 6 * self.MS_IN_DAY + 1,
+#                                            resolution=self.MS_IN_DAY)["data"]
+#         self.assertEqual(ret0[0]['value'], None)
+#         self.assertEqual(ret0[1]['value'], None)
+#         self.assertEqual(ret0[2]['value'], None)
+#         self.assertEqual(ret0[3]['value'], 131120)
+#         self.assertEqual(ret0[4]['value'], 179017)
+#         self.assertEqual(ret0[5]['value'], 3598)
+        
+    def test_final_andrew(self):
```

Comment:
name should be something like test_call_degree so it's clear what feature this is